src/docs/release.md

```diff
@@ -230,6 +230,7 @@ Right now that's:
 - pantsbuild.pants.contrib.errorprone
 - pantsbuild.pants.contrib.codeanalysis
 - pantsbuild.pants.contrib.jax_ws
+- pantsbuild.pants.contrib.mypy
```

Comment:
Would you mind adding confluence as well? Once #5340 lands it should be accurate.

Reply:
Done.